compiler/luci-interpreter/src/CMakeLists.txt

```diff
@@ -3,12 +3,14 @@ add_subdirectory(kernels)
 
 set(SOURCES
     "${LUCI_INTERPRETER_INCLUDE_DIR}/luci_interpreter/Interpreter.h"
+    GraphLoader.h
+    GraphLoader.cpp
     Interpreter.cpp
     KernelBuilder.h
     KernelBuilder.cpp
-    KernelMap.h
-    RuntimeToIR.h
-    TensorMap.h)
+    ModuleLoader.h
```

Comment:
I was thinking about naming.
`KernelBuilder` and `ModuleLoader`/`GraphLoader` are essentially on the same level of abstraction.
Maybe stick to one naming convention, for example `Builder` (I don't like loader, because we do not load anything here)